tests/benchmark/Dockerfile.in

```diff
@@ -1,6 +1,15 @@
-FROM ${FULL_BASE_IMAGE}
+FROM gcr.io/google-appengine/python:latest
 
-RUN hg clone -r 9923b81a1d34 https://hg.python.org/benchmarks /app/benchmarks
-WORKDIR /app/benchmarks
-RUN python perf.py -r -b default /usr/bin/python2.7 /usr/bin/python3.4
-RUN python perf.py -r -b default /usr/bin/python3.4 /opt/python3.5/bin/python3.5
+# Install performance
+RUN python2 -m pip install performance
+
+# Create virtual environment
+RUN python2 -m pip install -U virtualenv
+
+# Download ensurepip module which prevents the failure when benchmarking python3
+RUN wget https://www.python.org/ftp/python/3.4.2/Python-3.4.2.tgz
+RUN tar xzf Python-3.4.2.tgz
+RUN cp -R Python-3.4.2/Lib/ensurepip /usr/lib/python3.4
+
+# Run the benchmark and compare the performance, add the --debug-single-value flag to let the benchmark run in fastest mode
+RUN pyperformance run --debug-single-value --python=python2.7 -o py2.7.json && pyperformance run --debug-single-value --python=python3.4 -o py3.4.json && pyperformance run --debug-single-value --python=python3.5 -o py3.5.json && pyperformance compare py2.7.json py3.4.json --output_style table && pyperformance compare py3.4.json py3.5.json --output_style table
```

Comment:
I have just break them to multiple RUN steps and it works well! But it was failing yesterday for the reason that json file not found...I don't know why it just works today. Anyway, I have changed the code to break them up to separate lines.